sklearn/manifold/_mds.py

```diff
@@ -424,6 +426,18 @@ class MDS(BaseEstimator):
     (100, 2)
     """
 
+    _parameter_constraints = {
+        "n_components": [Interval(Integral, 1, None, closed="left")],
+        "metric": ["boolean"],
+        "n_init": [Interval(Integral, 1, None, closed="left")],
+        "max_iter": [Interval(Integral, 1, None, closed="left")],
+        "verbose": [Interval(Integral, 0, None, closed="left")],
```

Comment:
As of 480917f4de29e6ec084e14f4b1d174347ea11ca3 there is now the special `"verbose"` helper constraint, so this can be replaced.
```suggestion
        "verbose": ["verbose"],
```